modules/preview-react/segmented-control/lib/SegmentedControlItem.tsx

```diff
@@ -52,147 +53,196 @@ export interface ItemProps extends ButtonContainerProps {
   tooltipProps?: Omit<TooltipProps, 'children'>;
 }
 
-const getMinWidthStyles = (children: React.ReactNode, size: ButtonSizes) => {
-  switch (size) {
-    case 'large':
-      return children ? '112px' : '48px';
-    case 'medium':
-      return children ? '96px' : space.xl;
-    case 'small':
-      return children ? space.xxxl : space.l;
-    case 'extraSmall':
-      return children ? 'auto' : space.m;
-    default:
-      return children ? '96px' : space.xl;
-  }
-};
+export const segmentedControlItemStencil = createStencil({
+  extends: buttonStencil,
+  base: {
+    ...system.type.subtext.large,
+    fontWeight: system.fontWeight.bold,
+    textAlign: 'left',
+    borderRadius: system.shape.x1,
+    gap: system.space.x1,
 
-const getButtonSize = (size: ButtonContainerProps['size']) => {
-  switch (size) {
-    case 'large':
-      return 'medium';
-    case 'medium':
-      return 'small';
-    case 'small':
-      return 'extraSmall';
-    default:
-      return 'medium';
-  }
-};
+    [buttonColorPropVars.default.background]: system.color.bg.alt.soft,
+    [buttonColorPropVars.default.border]: system.color.border.transparent,
+    [buttonColorPropVars.default.icon]: system.color.static.gray.strong,
+    [buttonColorPropVars.default.label]: system.color.static.gray.strong,
```

Comment:
Curious if you're using `buttonColorPropVars` to still support the `colors` prop. I think in other areas we just use the stencil. So in this case this would use `[buttonStencil.vars.background]`

Reply:
I feel like it's good to use stencil here, as we can possible decide to remove `colors` props in the future. I try to keep consistent in `BaseButton`build-on usage, we have the same pattern in all variants of `Button`

Reply:
yeah let's use the `buttonStencil` instead. That's how we allow users to override styles

Reply:
Especially since we're extending the stencil as well